components/src/atoms/description-list/DescriptionList.html

```diff
@@ -0,0 +1,16 @@
+<dl class="list list-description">
+  <!--@slot Custom list content (Named slots)-->
+  <slot name="content">
+    <template v-for="item in elements">
+      <dt
+        :key="item.id"
+        class="list-description__label"
+      >
+        {{ item.label }}
+      </dt>
+      <dd class="list-description__value">
+        {{ item.value }}
+      </dd>
+    </template>
+  </slot>
+</dl>
```

Comment:
Can we put this entire component into `List` as an another story? I think you can move `.list-description` styles to `list.scss` as a next modifier `&--description`.
In stories it will something like
```
<a-list 
  tag="dl"
  class="list--description"
>
  <template v-for="item in items">
    <a-list-item
      tag="dt"
      class="list-description__label"
    >
      {{ item.label }}
    </a-list-item>
    <a-list-item
      tag="dd"
      class="list-description__value"
    >
      {{ item.value }}
    </a-list-item>
  </template>
</a-list>
```